psi4/src/psi4/libscf_solver/hf.cc

```diff
@@ -208,15 +199,22 @@ void HF::common_init() {
         }
     }
 
+    // Check that we have enough basis functions
+    for (int h = 0; h < nirrep_; ++h) {
+        if (std::max(nalphapi_[h], nbetapi_[h]) > nsopi_[h]) {
+            throw PSIEXCEPTION("Not enough basis functions to satisfy requested occupancies");
```

Comment:
```
             throw PSIEXCEPTION("Not enough linearly independent basis functions to satisfy requested occupancies");
```

Reply:
Proposed rewording is misleading. This exception is thrown due to a comparison against `nsopi`, which is the number of basis functions _regardless of_ linear dependence checking. (Linear dependence checking hasn't even occurred at the time this method is called.)

I suspect but have not confirmed that a system with enough basis functions, but not enough linearly independent basis functions, would escape this check and crash Psi. If true, we should file an issue.

Reply:
OK thanks for the clarification; I assumed that the number of symmetry functions was done *after* the linear dependence check.